community-operators/redis-enterprise/redisoperator.0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,153 @@
+      mediatype: image/png
+  links:
+    - name: Documentation
+      url: 'https://github.com/RedisLabs/redis-enterprise-k8s-docs'
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: true
+  - type: MultiNamespace
+    supported: true
```

Comment:
Is this operator actually configured to watch namespaces? The operator is going to be given a string that contains a list of namespaces to watch and the operator itself needs to create watches in each namespace.